security_group.tf

```diff
@@ -0,0 +1,27 @@
+# Security group for instances that run VENOM agents (Tainium and
```

Comment:
Typo (I think I have seen this misspelling elsewhere, but not sure if it was in code or GitHub issues):

```suggestion
# Security group for instances that run VENOM agents (Tanium and
```